lib/auth/tls_test.go

```diff
@@ -1274,8 +1274,11 @@ func (s *TLSSuite) TestWebSessions(c *check.C) {
 	user := "user1"
 	pass := []byte("abc123")
 
-	_, _, err = CreateUserAndRole(clt, user, []string{user})
+	newUser, err := CreateUserRoleAndRequestable(clt, user, "test-request-role")
 	c.Assert(err, check.IsNil)
+	c.Assert(newUser.GetRoles(), check.HasLen, 1)
```

Comment:
revised

I was initially hesitant to check for expected b/c:
1) I didn't think it was important that this test checks it (not that it would hurt)
2) The initial role name is assigned by a helper function, and is the `user` appended with `userName` = `user:userName`. This helper function could potentially change how initial roles are named, which can break the test needlessly. But, I also didn't think that would be likely.